src/Perlang.ConsoleApp/Perlang.ConsoleApp.csproj

```diff
@@ -37,7 +37,7 @@
 
     <ItemGroup>
         <PackageReference Include="ReadLine" Version="2.0.1" />
-        <PackageReference Include="System.CommandLine" Version="2.0.0-beta1.20158.1" />
+        <PackageReference Include="System.CommandLine" Version="2.0.0-beta1.21216.1" />
```

Comment:
There's an even newer (beta) version we could pull in [mentioned here](https://github.com/dotnet/command-line-api/pull/1271#issuecomment-834739025), but I don't want to do it in this PR since I'm not 100% sure that its current tests would illustrate whether the fix solves our problem or not. Postponing until we are ready with #190 and #191 and in the process of getting #188 merged; that one should have all the supporting code for properly testing the fix.

Reply:
For reference, I'm suspecting this is the dependency change which introduced the bug described in #199. More details can be found here: https://github.com/perlang-org/perlang/issues/199#issuecomment-894684410